extractor/src/main/java/org/schabi/newpipe/extractor/stream/StreamInfo.java

```diff
@@ -113,235 +119,210 @@ private static StreamInfo extractImportantData(StreamExtractor extractor) throws
         return new StreamInfo(serviceId, url, originalUrl, streamType, id, name, ageLimit);
     }
 
-    private static StreamInfo extractStreams(StreamInfo streamInfo, StreamExtractor extractor)
+    @Nonnull
+    private static StreamInfo extractStreams(final StreamInfo streamInfo,
```

Comment:
Since you are not calling and assigning this anymore (line 78), return void

Reply:
Done, I did this also for the `extractOptionalData` method